### DISCUSSION TITLE
cannot find a specification of parameter r_collect_cycle

### DISCUSSION THREAD

**Discussion by Marty56** [mampfes/hacs_waste_collection_schedule#3241] [Q&A]
Hi,

I am using AWM Munich.
In the examples the parameter r_collect_cycle is set to "1/2;G".
"1/2" could mean not weekly but biweekly.
But I have no idea what ";G" stands for.

Please clarify

**Comment by 5ila5** [comment]
from the AWM muenchen documentation:

> ## How to get the optional configuration arguments
> 
>  - Setup the component without the optional parameter and restart Home Assistant
>  - Check the Home Assistant log for entries from this component.
>  - The available options are listed in the error message.
>  - Adjust the configuration and restart Home Assistant.


For GUI setup you don't need to restart HA, but just leave it empty and look at the error message at the top of the configuration screen.

G is `Gerade Woche` I think

But if you don't see an additional screen (see below) when entering your address here https://www.awm-muenchen.de/entsorgen/abfuhrkalender you don't need to provide any collect_cycle

![grafik](https://github.com/user-attachments/assets/55e66c90-d0bf-4024-aba0-c378961171f8)
